package.json

```diff
@@ -18,39 +18,43 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
-    "broccoli-jscs": "2.0.0",
-    "ember-cli": "~2.8.0",
+    "broccoli-asset-rev": "^2.4.5",
+    "broccoli-jscs": "1.4.1",
+    "ember-ajax": "^2.4.1",
+    "ember-cli": "2.11.1",
     "ember-cli-app-version": "^2.0.0",
-    "ember-cli-blanket": "0.9.4",
-    "ember-cli-content-security-policy": "0.5.0",
-    "ember-cli-dependency-checker": "^1.1.0",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
-    "ember-cli-ic-ajax": "0.2.4",
-    "ember-cli-inject-live-reload": "^1.3.1",
+    "ember-cli-content-security-policy": "0.6.0",
+    "ember-cli-dependency-checker": "^1.3.0",
+    "ember-cli-ic-ajax": "1.0.0",
+    "ember-cli-htmlbars-inline-precompile": "^0.3.6",
+    "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-pretender": "0.7.0",
-    "ember-cli-qunit": "2.2.1",
+    "ember-cli-qunit": "^3.0.1",
     "ember-cli-release": "^0.2.9",
-    "ember-cli-sri": "2.1.1",
-    "ember-cli-test-loader": "1.1.0",
+    "ember-cli-shims": "1.0.2",
+    "ember-cli-sri": "^2.1.0",
+    "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-data": "~2.8.0",
-    "ember-disable-prototype-extensions": "^1.0.0",
+    "ember-data": "2.11.1",
+    "ember-disable-prototype-extensions": "^1.1.0",
     "ember-disable-proxy-controllers": "^1.0.1",
-    "ember-export-application-global": "^1.0.4",
-    "ember-font-awesome": "~2.1.1",
+    "ember-export-application-global": "^1.0.5",
+    "ember-font-awesome": "3.0.3",
+    "ember-load-initializers": "^0.6.0",
+    "ember-resolver": "^2.0.3",
     "ember-invoke-action": "1.3.0",
     "ember-proxy-util": "0.0.6",
     "ember-railio-formatting": "0.0.10",
-    "ember-suave": "2.0.1",
-    "loader.js": "^4.0.0"
+    "ember-suave": "4.0.0",
+    "ember-source": "~2.11.1",
+    "loader.js": "^4.0.10"
   },
   "keywords": [
     "ember-addon"
   ],
   "dependencies": {
-    "ember-cli-babel": "5.1.10",
-    "ember-cli-htmlbars": "1.1.0",
+    "ember-cli-babel": "5.1.7",
```

Comment:
Downgrade ==> Don't just say YES on each diff, but change it yourself step by step